foodie.rb

```diff
@@ -0,0 +1,91 @@
+class Order
+  attr_accessor :items, :oid, :cust_nm, :status 
```

Comment:
why there is order.rb file ? this and order.rb has same code which one is final version?

remove unwanted file. Add to readme about how to run foodie program.